c10/core/DispatchKey.cpp

```diff
@@ -128,6 +128,9 @@ const char* toString(DispatchKey t) {
     case DispatchKey::Autocast:
       return "Autocast";
 
+    case DispatchKey::AutocastCPU:
+      return "AutocastCPU";
+
```

Comment:
Any reason not to add `AutocastCUDA` at the same time?

Reply:
As far as I understand `Autocast` is the alias for `AutocastCUDA`, so `toString(AutocastCUDA) -> "Autocast"` and only one can be visible, either "Autocast" or "AutocastCUDA" and thus potentially `toString(Autocast) -> "AutocastCUDA"`. @bhosmer if you think it would be better like that, i'd be happy to add it as well.

Reply:
Ah! No, I didn't realize the two were aliased - thanks for the rundown 😁  given the [comment in DispatchKey.h](https://github.com/pytorch/pytorch/blob/master/c10/core/DispatchKey.h#L256-L257), probably best to leave it as-is until `AutocastCUDA` is truly CUDA-specific. (Plus backward compatibility I guess, though hopefully nobody's depending on these programmatically)

But it might be useful to add a comment in the `DispatchKey::Autocast` case at line 128, if you wouldn't mind. Maybe something like
```
// Note: AutocastCUDA and Autocast are the same, currently. See comments in DispatchKey.h
```